CMakeLists.txt

```diff
@@ -35,6 +37,8 @@ elseif(UNIX)
             DESTINATION "${CMAKE_INSTALL_DATADIR}/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/")
     install(FILES ${PROJECT_SOURCE_DIR}/onepinopenscpkcs11.json
             DESTINATION "${CMAKE_INSTALL_LIBDIR}/mozilla/pkcs11-modules")
+    install(FILES ${PROJECT_SOURCE_DIR}/idemiapkcs11.json
```

Comment:
The PKCS#11 module in question is for/from the "awp" productline. Maybe not call it as generic as "companyname" as there might very well be more modules from a company than one.

Reply:
As far as I know they distribute same packge to all customers.
Rename to AWP then?